test/library/encryption/e_log.go

```diff
@@ -0,0 +1,58 @@
+package encryption
+
+import (
+	"fmt"
+	"os"
+	"testing"
+)
+
+// E is like testing.T except it overloads some methods to print to stdout
```

Comment:
don't tell me that `go` is not an `OOP` language

Reply:
Go is not an OOP language.